test/conftest.py

```diff
@@ -0,0 +1,57 @@
+import pytest
```

Comment:
This is pytest feature . It automatically imports these fixtures to be used at multiple tests, and can be reused in other tests or fixtures.

Comprehensive docs for the same : https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
cool!